tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_generators/continuous_contact_check_process_generator.h

```diff
@@ -68,7 +68,7 @@ class ContinuousContactCheckProcessGenerator : public ProcessGenerator
 
   std::string name_;
 
-  double longest_valid_segment_length_{ 0.1 };
+  double longest_valid_segment_length_{ 0.005 };
```

Comment:
This seems pretty small.

Reply:
Yea, I will try 0.05 instead. The challenge I keep running into with continuous collision checking is that if you have two kinematic links that are always within say 2cm and are not disabled in ACM. Then if you move more than 2cm in a single time step it will report collision because the end state of the first shape overlaps with the start state of the second shape. This is why I have switched to using Discrete Continuous collision checking for motion planning and I am getting much better performance then I run the post check a fine resolution.

Reply:
Also I did not mean to include this in this PR. I will remove it and create a different PR to discuss further.

Reply:
I was actually running into the same thing. Do you think DISCRETE_CONTINUOUS or SINGLE_TIMESTEP should be the default? It is also more difficult to debug in this framework since the trajopt plotting has to sort of be hacked in there.

Reply:
Is `composite_profile->collision_constraint_config.type = trajopt::CollisionEvaluatorType::DISCRETE_CONTINUOUS;` all you have to do to set that? I'm still getting blue arrows

Reply:
> I was actually running into the same thing. Do you think DISCRETE_CONTINUOUS or SINGLE_TIMESTEP should be the default? It is also more difficult to debug in this framework since the trajopt plotting has to sort of be hacked in there.

I do believe we should set the default to Discrete Continuous. I think because of the previous issue it makes motion planning difficult to debug. I do think the post check is still good to use continuous but you need to do this at a fine resolution to no run into issues.

Reply:
> Is `composite_profile->collision_constraint_config.type = trajopt::CollisionEvaluatorType::DISCRETE_CONTINUOUS;` all you have to do to set that? I'm still getting blue arrows

You should still get the blue arrows with Discrete continuous. When interpolating between two states it stores the contact results with the deepest penetration and then shows the contact point on the start and end state like what is done with Continuous Collision checking.

Reply:
I reset it for this PR and will open up a new one changing these settings.

Reply:
I have create PR #392 to further discuss with proposed changes.